content/docs/deployment/csmoperator/modules/authorization.md

```diff
@@ -156,7 +155,7 @@ To enable reporting of trace data with [Zipkin](https://zipkin.io/), use the `cs
 
   Add the Zipkin values to the configMap:
   ```
-  ZIPKIN_URI: "http://PROXY_HOST:9411/api/v2/spans"
+  ZIPKIN_URI: "http://zipkin-addr:9411/api/v2/spans"
```

Comment:
What is 'zipkin-addr' here? We should explain/link to any variables if they are not defined in our yaml inputs

Reply:
Updated to say that ZIPKIN_ADDRESS is the IP or hostname of the Zipkin server. Using PROXY_HOST isn't correct.